app-guides/supercronic.html.md

```diff
@@ -82,10 +82,10 @@ Now that we've added `supercronic` to our `Dockerfile`, put the `crontab` at the
 $ fly deploy
 ```
 
-Then we'll need to scale the processes so that we only run one virtual machine container with the `cron` process. Be sure to change `web` to whatever number you had before.
+Then we'll need to scale the processes so that we only run one virtual machine container with the `cron` process.
 
 ```
-$ fly scale count cron=1 web=
+$ fly scale count -g cron 1
```

Comment:
This will work to scale one process group, but to scale more than one process group in a single command you would use the `fly scale count cron=1 web=2` for example.

Reply:
This was the error I got when running the example command.

```
❯ fly scale count cron=1 web=2
Error: --process-group flag is required when no group named 'app' is defined
```

So I thought maybe the command was out-of-date. Maybe the entire page should change `web` to `app` since that seems to be the new convention?

Reply:
Oh wow, yeah, that's not right. I'll look into things and then we'll see whether it's docs or the command that gets fixed. :)